### DISCUSSION TITLE
#29. VATZ biweekly meeting at 2023-03-15 08:00:00 -05:00:00

### DISCUSSION THREAD

**Discussion by xellos00** [dsrvlabs/vatz#414] [General]
### 1. Overall
<img width="1511" alt="image" src="https://user-images.githubusercontent.com/6308023/219247150-66cff1c7-9b59-4a55-ad58-6f9a733d61f9.png">

### 2. Statistic Rate
Sprint | Issue fulfillment | progress rate(%)
--: | :--: | :--:
27 | 2/19 | 10
28 | 6/25 |  24
---

@here
Please, Record current status for individual plugins 

---

<img width="1221" alt="image" src="https://user-images.githubusercontent.com/6308023/225156016-97fe7e2e-f3e3-4036-b963-74923d11e6b8.png">

**Comment by xellos00** [comment]
# 동요

- https://github.com/dsrvlabs/vatz/issues/410
- https://github.com/dsrvlabs/vatz/issues/323
- https://github.com/dsrvlabs/vatz/issues/388

---

Private/Public 
- https://github.com/dsrvlabs/vatz-plugin-hyperlane
- https://github.com/dsrvlabs/vatz-plugin-flow
- https://github.com/dsrvlabs/vatz-plugin-near

**Comment by rootwarp** [comment]
# 준교

- Public
  - https://github.com/dsrvlabs/vatz-plugin-matic
- Private
N/A

**Comment by gnongs** [comment]
# 진홍
- Private
  - https://github.com/dsrvlabs/vatz-plugin-relayer

**Comment by kim201212** [comment]
# 형기

- Private
   - https://github.com/dsrvlabs/vatz-plugin-eth-watcher

**Comment by xellos00** [comment]
@hqueue 
- Go Public/Private by Development or Company level
- Process to get a private repo to Public in list 
   - conduct how to upload their developed plugin into VATZ's list of plugins.

**Comment by xellos00** [comment]
@heejin-github 
Create plugin 
- https://github.com/dsrvlabs/vatz-plugin-sysutil, 
   - Conduct metric in the plugins
   
--- 
   
- Comment into README.md where to plugin feature request 
   - VATZ
   - Official Plugins

**Comment by xellos00** [comment]
Create a issue to deal with @heejin-github 

<img width="943" alt="image" src="https://user-images.githubusercontent.com/6308023/225336047-d96d84ca-8d73-4949-9bd6-dc4c28eb5233.png">
